### DISCUSSION TITLE
Front end for the registration.

### DISCUSSION THREAD

**Discussion by Bashamega** [is-a-dev/register#14073] [Ideas]
Hello:)
I think some people(including me) would benefit from being able to register for a subdomain directly from the site.
Also because of bots and malicious pages we can send the data to an email or store it temporarily, or even make a discord bot for it. it would be good for non GitHub users and i think it can make the process a little faster.
If you are going to implement this idea, i would like to be a part of it

**Comment by iostpa** [comment]
We've had a manage site and discord bot before, but we recently shut down the manage site due to many invalid PRs since it was created. Though we still have the discord bot for use in our discord server.

**Reply by Bashamega** [reply]
> We've had a manage site and discord bot before, but we recently shut down the manage site due to many invalid PRs since it was created. Though we still have the discord bot for use in our discord server.

Okay

**Comment by MaskDuck** [comment]
> bots and malicious pages we can send the data to an email or store it temporarily

wdym about this? regardless of any methods of registration your email is clearly visible

> make a discord bot for it

already exist, check our Discord server

> it would be good for non GitHub users

it doesn't harm you to register a GitHub account. and by the way we are is-a.**dev**, so basically when you register a domain you are a dev, therefore you should have a github account :) common sense

> it can make the process a little faster

no, we've had a website before, but it turned out to be chaos since our maintainers had to review a massive amount of invalid PRs which waste our time.

**Reply by wdhdev** [reply]
This. The amount of hours of my time that has been wasted on idiots registering invalid domain names is insane. The amount of invalid PRs that have been registered through the manage site is probably in the thousands.
